docfx.json

```diff
@@ -476,7 +477,11 @@
         "docs/standard/security/**.md": "dotnet-security",
         "docs/visual-basic/language-reference/error-messages/**/**.md": "vb-diagnostics",
         "docs/visual-basic/misc/**/**.md": "vb-diagnostics"
-      },
+      },    
+     "recommendations": { 
+       "architecture/**/**.md": false,
+       "standard/design-guidelines/**/**.md": false  
+     },
```

Comment:
We should add the language standards here:

```suggestion
        "_csharplang/spec/*.md": false,
        "_vblang/spec/*.md": false
     },
```

Reply:
They are marked as "language-reference", so they will automatically be excluded. It probably doesn't hurt to be explicit, but then are there any other reference folders we want to explicitly exclude?

```
      "ms.topic": {
        "_csharplang/**/*.md": "language-reference",
        "_vblang/spec/*.md": "language-reference",
```

Reply:
Those are the only ones I'd want to be explicit on. If recommendations eventually runs on reference content, I can see value in having it in other areas, but I'd still want these off.

Reply:
Since those cross-repo refs map to the csharp/language-reference and visual-basic/reference/language-specification folders, do we use the source path or the destination path here? I would guess the destination.

```
        "src": "_csharplang/spec",
        "dest": "csharp/language-reference/language-specification",

        "src": "_vblang/spec",
        "dest": "visual-basic/reference/language-specification",
```

Reply:
I couldn't tell from the docs on the feature. You're likely right.

Reply:
Actually, I think I'm wrong based on other metadata you've set on those paths. Your initial suggestion was correct.

![image](https://user-images.githubusercontent.com/24882762/118333688-fd38a300-b4c0-11eb-817a-7319b14008fc.png)